{"name": "test003", "status": "failed", "statusDetails": {"message": "AssertionError: assert -4 == 5", "trace": "self = <testDemo.test002.TestClass object at 0x7fbf44d4dac0>\n\n    @allure.feature('allure测试')\n    @allure.story('test003')\n    def test003(self):\n        c = Calc()\n        a = c.jian(2, 6)\n>       assert a == 5\nE       assert -4 == 5\n\ntest002.py:34: AssertionError"}, "attachments": [{"name": "stdout", "source": "5b729085-d102-4cf4-9f08-b65e3ac9aecb-attachment.txt", "type": "text/plain"}], "start": 1639302523196, "stop": 1639302523196, "uuid": "6647ba7d-84a0-47b0-b308-e85fd105ae0e", "historyId": "6db8c25dc9c2ab85bc85f75e5e68df97", "testCaseId": "b1bd6b7b218c9d4e2c74197b2572f536", "fullName": "test002.TestClass#test003", "labels": [{"name": "feature", "value": "allure测试"}, {"name": "story", "value": "test003"}, {"name": "suite", "value": "test002"}, {"name": "subSuite", "value": "TestClass"}, {"name": "host", "value": "MBP.local"}, {"name": "thread", "value": "38789-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test002"}]}